src/partials/header-content.hbs

```diff
@@ -19,34 +19,63 @@
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor-reveal.js::index.adoc')}}">Asciidoctor reveal.js</a>
+            --}}
+            <a class="navbar-item" href="#">Asciidoctor reveal.js</a>
+          </div>
+        </div>
+        <div class="navbar-item has-dropdown is-hoverable">
+          <div class="navbar-link">Tooling</div>
+          <div class="navbar-dropdown">
+            <div class="navbar-item"><strong>Build Plugins</strong></div>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'maven-plugin::index.adoc')}}">Maven Tools</a>
+            <a class="navbar-item" href="#">Gradle Plugin</a>
+            {{!--
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoclet::index.adoc')}}">Asciidoclet <small>Javadoc</small></a>
+            --}}
+            <a class="navbar-item" href="#">Asciidoclet <small>Javadoc</small></a>
             <hr class="navbar-divider">
-            <div class="navbar-item"><strong>AsciidoctorJ</strong></div>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctorj" target="_blank" rel="noopener">Source</a>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctorj/issues" target="_blank" rel="noopener">Issues</a>
+            <div class="navbar-item"><strong>Text Editors & Viewers</strong></div>
+            <a class="navbar-item" href="#">Browser Extension</a>
+            <a class="navbar-item" href="#">Atom Add-ons</a>
+            <a class="navbar-item" href="https://intellij-asciidoc-plugin.ahus1.de/docs" target="_blank" rel="noopener">IntelliJ Plugin</a>
           </div>
         </div>
         <div class="navbar-item has-dropdown is-hoverable">
-          <div class="navbar-link">Community</div>
+          <div class="navbar-link">Extensions</div>
           <div class="navbar-dropdown is-right">
-            <a class="navbar-item" href="http://discuss.asciidoctor.org" target="_blank" rel="noopener">Discuss</a>
-            <a class="navbar-item" href="https://gitter.im/asciidoctor/asciidoctor" target="_blank" rel="noopener">Chat - General</a>
-            <a class="navbar-item" href="https://gitter.im/asciidoctor/asciidoctor.js" target="_blank" rel="noopener">Chat - Asciidoctor.js</a>
-            <a class="navbar-item" href="https://gitter.im/asciidoctor/asciidoctorj" target="_blank" rel="noopener">Chat - AsciidoctorJ</a>
+            <div class="navbar-item"><strong>Extended Syntax</strong></div>
+            <a class="navbar-item" href="#">Asciidoctor Diagram</a>
+            <a class="navbar-item" href="#">Asciidoctor Chart</a>
+            <a class="navbar-item" href="#">Asciidoctor bibtex</a>
+            <div class="navbar-item"><strong>Feature Adapters</strong></div>
+            <a class="navbar-item" href="#">Asciidoctor Mathematical <small>stem</small></a>
```

Comment:
Proving my ignorance here! What's the difference between math and prism and the the other three?
Why not keep things simple and have them all under "Extensions"?

Reply:
Math and Prism build on existing syntax, namely `stem` and listing block but provide additional features (build-time rendering, different syntax highlighter engine...).

The Chart, BibTeX and Diagram extend the syntax to provide more capabilities. Managing bibliographical references, adding diagrams and charts...

Reply:
Exactly! Though, it isn't much of a concern right now since this will be commented out in the first version.